docs/siem/detections/api/exceptions/api-update-exception-item.asciidoc

```diff
@@ -0,0 +1,138 @@
+[[exceptions-api-update-item]]
+=== Update exception item
+
+Updates an existing exception item.
+
+==== Request URL
+
+`PUT <kibana host>:<port>/api/exception_lists/items`
+
+==== Request body
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
+
+|No, defaults to empty array.
+
+|`description` |String |Describes the exception item. |Yes
+|`entries` |<<entries-object-schema, entries[]>> |Array containing the
+exception queries. Boolean `AND` logic is used to evaluate the relationship
+between array elements. If you want to use `OR` logic, create a separate
+exception item. |No, defaults to an empty `entries` array.
```

Comment:
Same as with creation, `entries` can no longer be an empty array or `undefined`.